scripts/build

```diff
@@ -12,25 +12,23 @@ cd $basedir
 
 if [[ "$OSTYPE" == "cygwin" || "$OSTYPE" == "msys" ]]; then
     # Patch version.h
-    if [[ -n "$OPENRCT2_BUILD" ]]; then
-        echo -e "\033[0;36mPatching version.h...\033[0m"
-        fileversion=$OPENRCT2_VERSION.$OPENRCT2_BUILD
-        productversion="$fileversion-${OPENRCT2_SHA1_SHORT}"
-        fileversion=${fileversion//./,}
-        # FILEVERSION in the resource file can only take up to 4 digits in the version string, so we remove the surplus of version numbers
-        fileversion=$(echo $fileversion | cut -f1-4 -d",")
-        echo "#define OPENRCT2_FILE_VERSION $fileversion" > "resources/version.h"
-        echo "#define OPENRCT2_PRODUCT_VERSION \"$productversion\"" >> "resources/version.h"
-        cat "resources/version.h"
-    fi
+    echo -e "\033[0;36mPatching version.h...\033[0m"
+    fileversion=$OPENRCT2_VERSION.$OPENRCT2_BUILD
+    productversion="$fileversion-${OPENRCT2_SHA1_SHORT}"
+    fileversion=${fileversion//./,}
+    # FILEVERSION in the resource file can only take up to 4 digits in the version string, so we remove the surplus of version numbers
+    fileversion=$(echo $fileversion | cut -f1-4 -d",")
+    echo "#define OPENRCT2_FILE_VERSION $fileversion" > "resources/version.h"
+    echo "#define OPENRCT2_PRODUCT_VERSION \"$productversion\"" >> "resources/version.h"
+    cat "resources/version.h"
 
     # Build everything
     echo -e "\033[0;36mBuilding OpenRCT2 for Windows $CONFIGURATION|$PLATFORM...\033[0m"
     vstool msbuild openrct2.proj -t:build -p:Breakpad=true -p:UsePCH=true
 
     # Create openrct2.exe and openrct2.com with correct subsystem
     cp bin/openrct2.exe bin/openrct2.com
```

Comment:
`openrct2.exe` is copied to `openrct2.com`, but then not modified to be a console subsystem.
I see you have moved the openrct2.com creation to the signing process, but doesn't this mean dev builds no longer have a correct version of openrct2.com?

Reply:
There's no need for the commit that modified it. I left it when I was debugging the signing process, but it works even on files named `.com`